README.rst

```diff
@@ -425,6 +425,32 @@ view (only when using HTTP POST).
 The signal is emitted just before a successful response is returned,
 it is not sent if the response was not successful.
 
+Notifications
+-------------
+
+OpenWISP Monitoring uses `OpenWISP Notifications <https://github.com/openwisp/openwisp-notifications>`_ 
+for managing notifications. OpenWISP Notifications provides ``notify`` signal which simplifies generation
+of notifications. You can learn more about creating notifications from `documentaion of OpenWISP Notifications
+<https://github.com/openwisp/openwisp-notifications/tree/master#generating-notifications>`_.
```

Comment:
You have to understand most users are users and not developers, they won't care about how to send notifications because they will not touch the code, some of them will, but that should be treated later.

First of all we have to inform them about what is available, and it should be done in a very concise way.

Please refactor the whole changes to the readme so that we have a structure like the following:

```
Default notifications / alerts
------------------------------------

<notification types here>
```

Then in the section about how to extend openwisp-monitoring, in the end, add:

```
Registering new notification types
---------------------------------------------

Link docs in openwisp-notifications.

Sending notifications
----------------------------

Specify this needs to be done when a new notification type is registered, link the readme of openwisp-notifications.
```